*** ./tests/A-01-06-M-ok.out	2022-10-01 08:05:11.702763012 +0100
--- ./tests/A-01-06-M-ok.outhyp	2022-10-10 13:58:43.502355339 +0100
***************
*** 6,13 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro wqrqwrwqqw
! MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
  3 - Gestão de clientes
--- 6,12 ----
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
  3 - Gestão de clientes
